docs: Update Redis connection string instructions for Railway (#10246)
medusajs/medusa: www/apps/resources/app/deployment/medusa-application/railway/page.mdx

- `ADMIN_CORS`'s value is the URL of the admin dashboard. If you don’t have it yet, or you’re deploying the admin with the Medusa application, you can skip adding it for now.
- `AUTH_CORS`'s value is the URLs of any application authenticating users, customers, or other actor types, such as the storefront and admin URLs. The URLs are separated by commas. If you don’t have the URLs yet, you can set its value later.
- Change `DISABLE_MEDUSA_ADMIN` to `false` if you’re hosting the admin with the Medusa application.

> - `REDIS_URL`'s value is automatically generated using Railway’s template syntax. Please note that Railway does not support IPv4 private networking, so make sure to add the `?family=0` to the connection string to support both IPv6 and IPv4 connections.

Reviewer: ```suggestion
- `REDIS_URL`'s value is automatically generated using Railway’s template syntax. You add the `?family=0` to the connection string to support both IPv6 and IPv4 connections.
```

Reviewer: Also can you add the same line in the section below for worker mode environment variables?
